app/views/products/show.html.erb

```diff
@@ -1,6 +1,6 @@
 <p style="color: green"><%= notice %></p>
 
-<%= render @product %>
+<%= render partial: @product, locals: { one_image: false } %>
```

Comment:
what's the use of one_image?

Reply:
At product list, only one image is required to be shown and on particular product, all images are to be shown
Since both pages uses same partial, one_image option is passed to show either one or multiple images as desired